rpctest/README.md

```diff
@@ -0,0 +1,38 @@
+rpctest
+=======
+
+[![Build Status](https://travis-ci.org/btcsuite/btcd.png?branch=master)]
+(https://travis-ci.org/btcsuite/btcd)
+
+Package rpctests provides a btcd-specific RPC testing harness for excersing 
+regressions via the RPC interface.
+...TODO
+
+## Documentation
+
+[![GoDoc](https://godoc.org/github.com/btcsuite/btcd/rpctest?status.png)]
+(http://godoc.org/github.com/btcsuite/btcd/rpctest)
+
+Full `go doc` style documentation for the project can be viewed online without
+installing this package by using the GoDoc site
+[here](http://godoc.org/github.com/btcsuite/btcd/rpctest).
+
+You can also view the documentation locally once the package is installed with
+the `godoc` tool by running `godoc -http=":6060"` and pointing your browser to
+http://localhost:6060/pkg/github.com/btcsuite/btcd/rpctest
+
+
+## Installation
+
+```bash
+$ go get github.com/btcsuite/btcd/rpctest

```

Comment:
We've been trying to switch over to using the `-u` flag since it work for update as well as initial install.

`go get -u github.com/btcsuite/btcd/rpctest`